src/main/java/core/basesyntax/strategy/impl/OperationHandlerImplBalance.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationHandler;
+
+public class OperationHandlerImplBalance implements OperationHandler {
```

Comment:
the same for the strategy implementation name. Impl is always adding to the end class name. (check all strategy implementations😉)

```suggestion
public class BalanceOperationHandlerImpl implements OperationHandler {
```